testing/workflows/components/kfctl_go_test.jsonnet

```diff
@@ -33,7 +33,13 @@ local runPath = srcDir + "/testing/workflows/run.sh";
 local kfCtlPath = srcDir + "/bootstrap/bin/kfctl";
 local kubeConfig = testDir + "/kfctl_test/.kube/kubeconfig";
 
-// Name for the Kubeflow app.
+// cluster_creation_script specifies the script to run in order to create 
```

Comment:
Will we move to the argo_client/tekton_client rather than jsonnet  in a future PR ?

Reply:
Yes, the issue is https://github.com/kubeflow/kubeflow/issues/4189